addon/mixins/session-route.js

```diff
@@ -1,4 +1,14 @@
 import Mixin from '@ember/object/mixin';
 
 export default Mixin.create({
+  async model(params) {
```

Comment:
I think this is the default magic ember model implementation and this code may be unnecessary.

Reply:
you are correct. yanked it.